README.md

```diff
@@ -16,17 +14,22 @@ machine:
 
 dependencies:
   post:
-    - curl -L https://s3.amazonaws.com/codeclimate/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+    - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
     - chmod +x ./cc-test-reporter
 
 test:
   pre:
     - ./cc-test-reporter before-build
 
   post:
-    - ./cc-test-reporter after-build --exit-code $CIRCLE_EXIT_CODE
+    - ./cc-test-reporter after-build --exit-code $TEST_SUITE_EXIT_CODE
 ```
 
+  Where:
+
+  - `$TEST_EXIT_CODE` should be the exit code when the test suite ran. Some CI
+    services don't expose directly a var with the
```

Comment:
I probably wouldn't note it as optional FWIW, or discuss ways in which it may or may not be used.